_health-care/_js/components/form-elements/ErrorableTextInput.jsx

```diff
@@ -56,7 +62,7 @@ class ErrorableTextInput extends React.Component {
             id={this.inputId}
             placeholder={this.props.placeholder}
             type="text"
-            value={this.props.value}
+            value={this.props.value.value}
```

Comment:
I wonder if we should change the prop name `value` to be something else? Maybe `field` or `fieldData`? Since what used to be `value` is now a function that returns 2 different things, an actual `value` and `dirty`.